cli/src/lib.rs

```diff
@@ -147,6 +149,7 @@ pub fn run<W>(worker: W, version: cli::VersionInfo) -> error::Result<()> where
 		cli::ParseAndPrepare::RevertChain(cmd) => cmd.run_with_builder::<(), _, _, _, _, _>(|config|
 			Ok(service::new_chain_ops(config)?), load_spec),
 		cli::ParseAndPrepare::CustomCommand(PolkadotSubCommands::ValidationWorker(args)) => {
+			#[cfg(not(target_os = "unknown"))]
```

Comment:
```suggestion
			#[cfg(not(feature = "browser"))]
```

However I would instead print an error, rather than just exiting silently.